src/main/java/liquibase/sdk/maven/plugins/BuildPluginXmlMojo.java

```diff
@@ -0,0 +1,199 @@
+package liquibase.sdk.maven.plugins;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.SortedMap;
+import java.util.SortedSet;
+
+@Mojo(name = "build-plugin-xml")
+public class BuildPluginXmlMojo extends AbstractMojo {
+
+    @Parameter(required = true)
+    private File liquibaseClassesDir;
+
+    @Parameter(required = false)
+    private File liquibaseProClassesDir;
+
+    @Parameter(required = true)
+    private File outputFile;
+
+    @Override
+    public void execute() throws MojoExecutionException, MojoFailureException {
+        ClassLoader originalClassLoader = Thread.currentThread().getContextClassLoader();
+        try {
+            if (!liquibaseClassesDir.exists()) {
+                throw new MojoFailureException(liquibaseClassesDir.getAbsolutePath() + " does not exist");
+            }
+
+            Document pluginXml = DocumentBuilderFactory.newInstance().newDocumentBuilder().newDocument();
+            Element pluginElement = pluginXml.createElement("plugin");
+            pluginXml.appendChild(pluginElement);
+
+            addNode("name", "liquibase-maven-plugin", pluginElement, pluginXml);
+            addNode("description", "A Maven plugin wraps up some of the functionality of Liquibase", pluginElement, pluginXml);
```

Comment:
Probably should change the description now:

```suggestion
            addNode("description", "A Maven plugin wraps up the functionality of Liquibase", pluginElement, pluginXml);
```